filters/crop/CropFilter.cpp

```diff
@@ -68,7 +69,11 @@ void CropFilter::addArgs(ProgramArgs& args)
     args.add("outside", "Whether we keep points inside or outside of the "
         "bounding region", m_cropOutside);
     args.add("a_srs", "Spatial reference for bounding region", m_assignedSrs);
-    args.add("bounds", "Bounds box for cropped points", m_bounds);
+    args.add("bounds", "Point box for cropped points", m_bounds);
+    args.add("point", "Crop within 'distance' from a 2D or 3D point", m_points).
+        setErrorText("Invalid point specification must be in the form \"(1.00, 1.00)\""
```

Comment:
If we do the WKT/GeoJSON thing as proposed below, we can't really do the combined thing proposed here. There were no ordering issues because the points are all buffered by the same distance.